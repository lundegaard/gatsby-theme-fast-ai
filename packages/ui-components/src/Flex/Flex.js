import React from 'react';
import { Flex as RebassFlex } from 'rebass';

const Flex = props => <RebassFlex {...props} />;

export default Flex;
