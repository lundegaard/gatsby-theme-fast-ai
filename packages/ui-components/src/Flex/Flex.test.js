import React from 'react';
import { shallow } from 'enzyme';

import Flex from './Flex';

describe('<Flex />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Flex {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
