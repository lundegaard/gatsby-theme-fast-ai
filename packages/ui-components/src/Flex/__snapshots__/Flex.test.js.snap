// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex /> matches snapshot when shallowly rendered 1`] = `<Styled(div) />`;
